src/librbd/cache/pwl/ssd/WriteLog.h

```diff
@@ -136,6 +136,8 @@ class WriteLog : public AbstractWriteLog<ImageCtxT> {
                            Context *ctx);
   void aio_read_data_blocks(std::vector<WriteLogCacheEntry*> &log_entries,
```

Comment:
Is this function still needed? I don't think. The only other place it's used is[0][1]. In both places it `WriteLogCacheEntry` can be changed.

[0]https://github.com/ceph/ceph/blob/master/src/librbd/cache/pwl/ssd/WriteLog.cc#L960
[1]https://github.com/ceph/ceph/blob/master/src/librbd/cache/pwl/ssd/WriteLog.cc#L513

Reply:
@MahatiC  yes, the function should be removed.

Reply:
@MahatiC  this function shouldn't be removed, `aio_read_data_block()` keeps raw pointer. 

https://github.com/ceph/ceph/blob/master/src/librbd/cache/pwl/ssd/WriteLog.cc#L960 this line don't changed.

Reply:
Does it have to be a raw pointer?  Having two different rather long functions do exactly the same thing just on different types of pointers is really undesirable.

Is there a way to refactor this?